### DISCUSSION TITLE
Are there any plans for Atom editor support?

### DISCUSSION THREAD

**Discussion by MiracleBlue** [community/community#7158] [Copilot Conversations]
I'm a long time Atom user and have customised my Atom setup to the nth degree, which is great because my customisations have become an important part of my workflow in general.  I can't do the same sorts of things in VSCode so its very difficult to make the switch over.  I do hope you might consider adding support for Atom.  Please also accept my offering of one Owl emoji:

# 🦉

Everyone loves owls.  How could you say no to an owl?

**Comment by pieterhouwen** [comment]
Also would love to see this

**Comment by Perritu** [comment]
Can I see somewhere an API implementation docs so I can make a WIP with Atom-io editor?

**Reply by pieterhouwen** [reply]
These?
https://flight-manual.atom.io/api/
https://atom.io/packages/api-docs

I doubt GH will open-source their Copilot api, documentation is also a bit light on copilot

**Reply by Perritu** [reply]
I mean, I already knew about the way to build Atom plugins. What I was lookin' for is how to interact with GH's copilot api... Sadly, as you say, GH mostly won't open-source their copilot's api.

**Comment by alfuken** [comment]
It is rather a pity seeing neovim support but no atom support. Shame.
